src/pages/GoverningBody/Announcement/Form.module.css

```diff
@@ -0,0 +1,35 @@
+.cardButton {
+    display: flex;
+    width: 100%;
+    justify-content:flex-end;
+    float: right;
+}
+.cardButton>* {
+  margin-right: 8px;
+} 
+  .formField {
+    max-width: 370px;
+    min-width: 370px;
+  }
+  
+  .selectField {
+    max-width: 370px;
+    min-width: 370px;
+  }
+
+@media(max-width: 400px) {
+  .formField{ 
+    max-width: 100%;
+    min-width: 100%;
+  }
+  .selectField {
+    max-width: 100%;
+    min-width: 100%;
+  }
+}
+.buttons{
+  margin-right: 0px !important;
```

Comment:
Do you try it without !important?

Reply:
This is the way it's done in all project, doesn't work without this